nexus/db-queries/src/db/queries/next_item.rs

```diff
@@ -700,6 +1034,68 @@ mod tests {
         }
     }
 
+    #[derive(Debug, Clone, Copy)]
+    struct NextItemSelfJoinedQuery {
+        inner: NextItemSelfJoined<item::dsl::item, i32, item::dsl::value>,
+    }
+
+    // These implementations are needed to actually allow inserting the results
+    // of the `NextItemSelfJoinedQuery` itself.
+    impl NextItemSelfJoinedQuery {
```

Comment:
I know this is test-only code, but could we supply the bounds to `new()` as arguments? Took me a moment to realize this was bound to the range `[0, 9]`

Reply:
Added arguments in 75311d5a7